### PR TITLE
Finished implementing entry point structure

### LINKED ISSUE #22: Entry : Start working on entry-point
As in Title. Start working on the entry-point and the user experience coming with it.

### PR DESCRIPTION
**Pull request by SapphireNick**
The main.cc in the root folder is, for now, to keep track of how the user experience should look like and simultaneously keep track of the current ux structure.

Closes #22

### CODE REVIEW

File: include/Shadowheart.hh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    /**
+     * @brief Function to start actual game loop and execute load function
+     * @param: 
+     * @return: void
+     */
+    void start();
+
+} // namespace Shadowheart
+
+#endif // SHADOWHEART_HH
```
Comment (d-jakob) [inline include/Shadowheart.hh:36]: no newline at the end of file

File: main.cc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    return;
+}
+
+void Shadowheart::draw() {
+    return;
+}
+
+int main() {
+    Shadowheart::start();
+}
```
Comment (d-jakob) [inline main.cc:17]: no newline at the end of file

Comment (d-jakob) [inline main.cc:17]: function style see #21 ?

File: src/Shadowheart.cc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+        // User functions for drawing and game-logic goes here
+        Shadowheart::draw();
+        Shadowheart::update();
+    }
+
+    // Terminate everything on closure
+    glfwTerminate();
+    return;
+}
```
Comment (d-jakob) [inline src/Shadowheart.cc:54]: newline again

**Review by d-jakob — CHANGES_REQUESTED** [review]
Also we agreed on an indentation level of 2 spaces instead of the 4 in all of the files in this pr, right?

File: include/Shadowheart.hh (hunk 1 above)
Comment (SapphireNick) [reply include/Shadowheart.hh:36]: As said in my comment, formatting issues will be taken care of after review and before merging.

**Comment by SapphireNick** [comment]
Fixed formatting. The main.cc is again only to test and see, if the implementation of the entry point works as we anticipate. It is not a main part of the project.

**Review by d-jakob — APPROVED** [review]

**Pull request merged**